packages/core/src/fhir-deduplication/__tests__/group-same-conditions.test.ts

```diff
@@ -56,14 +57,13 @@ describe("groupSameConditions", () => {
     expect(snomedMap.size).toBe(1);
   });
 
-  it("does not lose conditions that have neither snomed nor icd-10 codes", () => {
+  it("loses conditions that have neither snomed nor icd-10 codes", () => {
```

Comment:
to clarify, can we also have a test that shows that the removeOtherCodes function works? I was a bit confused since there are two different distinct  behaviors here:
1. dont put conditions into the map that dont have either snomed or icd
2. for conditions that do have snomed or icd but also have epic or IMo code system, strip those codes from the condition

Reply:
already have both tests. I updated the descriptions, so they're easier to understand